src/Kernel/client/kernel.ts

```diff
@@ -67,7 +67,7 @@ function defineQSharpMode() {
         {
             // built-in magic commands
             token: "builtin",
-            regex: String.raw`(%(config|estimate|lsmagic|package|performance|simulate|toffoli|version|who|workspace))\b`,
+            regex: String.raw`(%(config|estimate|lsmagic|package|performance|simulate|toffoli|version|who|workspace|trace))\b`,
```

Comment:
Silly, but the rest of these are in alphabetical order, so:
```suggestion
            regex: String.raw`(%(config|estimate|lsmagic|lsopen|package|performance|simulate|toffoli|trace|version|who|workspace))\b`,
```
(I also snuck in one other missing one, if you don't mind including that...)

Reply:
Sure! I just didn't know if I should add it to this PR or another one straight into `master` haha. Thanks!

Reply:
Doesn't matter much; this way avoids any potential merge conflicts... 😊